.github/workflows/eudonet_paris_import.yml

```
@@ -0,0 +1,91 @@
+name: Eudonet Paris Import
+
+# on:
+#   schedule:
+#     - cron: '30 16 * * 1' # Voir https://crontab.guru/ : tous les lundis à 16h30
+
+on:
+  push:
+    branches:
+      - feat/eudonet-auto
```

À changer avant de merger

```suggestion
on:
  schedule:
    - cron: '30 16 * * 1' # Voir https://crontab.guru/ : tous les lundis à 16h30

```